diff -ruN orig/configure/CONFIG_SITE 3.15.5/configure/CONFIG_SITE
--- orig/configure/CONFIG_SITE	2016-12-13 14:25:23.000000000 +0100
+++ 3.15.5/configure/CONFIG_SITE	2020-07-31 19:33:28.679579528 +0200
@@ -136,6 +136,9 @@
 #  Must be either YES or NO.
 STATIC_BUILD=NO
 
+# Switch gcc to use c++11
+USR_CXXFLAGS_Linux += -std=c++11
+
 # Host build optimization.
 #  Must be either YES or NO.
 HOST_OPT=YES
