diff -uir open62541.orig/include/open62541/types.h open62541/include/open62541/types.h
--- open62541.orig/include/open62541/types.h    2024-08-09 11:27:52.000000000 +0200
+++ open62541/include/open62541/types.h 2024-09-03 10:58:38.813382735 +0200
@@ -1067,7 +1067,7 @@
  * If the member is an array, the offset points to the (size_t) length field.
  * (The array pointer comes after the length field without any padding.) */
 #ifdef UA_ENABLE_TYPEDESCRIPTION
-UA_Boolean
+UA_Boolean UA_EXPORT
 UA_DataType_getStructMember(const UA_DataType *type,
                             const char *memberName,
                             size_t *outOffset,
diff -uir open62541.orig/src/ua_types.c open62541/src/ua_types.c
--- open62541.orig/src/ua_types.c       2024-09-03 11:00:06.897318665 +0200
+++ open62541/src/ua_types.c    2024-09-02 13:27:49.688344016 +0200
@@ -1882,7 +1882,7 @@
 }

 #ifdef UA_ENABLE_TYPEDESCRIPTION
-UA_Boolean
+UA_Boolean UA_EXPORT
 UA_DataType_getStructMember(const UA_DataType *type, const char *memberName,
                             size_t *outOffset, const UA_DataType **outMemberType,
                             UA_Boolean *outIsArray) {
