--- orig/src/ua_types.c 2024-09-02 11:31:15.514006029 +0200
+++ 1.3/src/ua_types.c      2024-09-02 11:31:29.499010032 +0200
@@ -1882,7 +1882,7 @@
 }

 #ifdef UA_ENABLE_TYPEDESCRIPTION
-UA_Boolean
+UA_Boolean UA_EXPORT
 UA_DataType_getStructMember(const UA_DataType *type, const char *memberName,
                             size_t *outOffset, const UA_DataType **outMemberType,
                             UA_Boolean *outIsArray) {
